### ISSUE TITLE
[Feature Request]: Please add a setting to choose path separator

### ISSUE THREAD

**Issue by husainazkas** [davidquinn/psi-header#80]
I think it would be better if there is an option like `"pathSeparator"` with enum values `["default", "windows", "unix"]` for system variables like `filepath`, `fullpath`, `filerelativepath`, etc due to team-work with various device/OS.

**Comment by davidquinn** [comment]
I'll take a look at this soon

D

**Comment by davidquinn** [comment]
Do you mean that you want to force it to a specific path separator regardless of the OS?

**Comment by husainazkas** [comment]
yes. In my case, I use macOS and my teammate use Windows. Everytime I'm working on some files that my teammates already edited or vice versa, the `<<filerelativepath>>` will always changed because of the different path separator on each OS.
For example:

After I edit the file, the header that contains `<<filerelativepath>>` will change to this
```
* Filename     : /service/requests/requests-service.go
```
Then when my teammates edit the file, the header that contains <<filerelativepath>> will change to this
```
* Filename     : \service\requests\requests-service.go
```

**Comment by davidquinn** [comment]
OK, I can look to put in a setting that overrides the path separator in the final text.

D

**Comment by davidquinn** [comment]
this is now done, There is a new setting under `psi-header.config` called `overridePathSeparator`. Extension has been updated to v1.25.0. See README for details.

**Comment by husainazkas** [comment]
Cool! Thank you. I really appreciate!
